dashboard/app/views/levels/show.html.haml

```diff
@@ -1,3 +1,6 @@
+- content_for(:head) do
+  = javascript_include_tag 'js/code-studio-common'
```

Comment:
I think this is the scariest change in this PR - I'll make sure I run all of our UI tests because of this.  This pulls the common output from factor-bundle into every rendered level page (because it has to be included in order to use the other factor-bundle output) which, at least short-term, is likely to include some duplicated stuff in every page.